dipy/align/api.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import nibabel as nib
+
+from imwarp import SymmetricDiffeomorphicRegistration
+from imwarp import DiffeomorphicMap
+from metrics import CCMetric
+import vector_fields as vfu
+
+
+def syn_registration(moving, static, metric=CCMetric(3), 
```

Comment:
I have to run for a bit but in a couple of hours will also give you another clarifying example. See you soon! :)